frontend/cypress/integration/EmergencyDetailsPageTests/FilledEmergencyDetailsTests.spec.js

```diff
@@ -0,0 +1,112 @@
+/// <reference types="cypress" />
+
+context('Test Filled Emergency Details', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:3000/road-trip/1/emergency-details');
+    cy.intercept('GET', '/api/roadtrip/1', {
+      statusCode: 200,
+      body: {},
+    });
+    cy.intercept('GET', '/api/roadtrip/1/attendees', {
+      statusCode: 200,
+      body: ['a', 'b', 'c'],
+    });
+
+    cy.intercept('GET', '/api/roadtrip/1/map', {
+      statusCode: 200,
+      body: {},
+    });
+    cy.intercept('GET', '/api/roadtrip/1/isUserOrganiser', {
+      statusCode: 200,
+      body: { result: true },
+    });
+    cy.intercept('GET', '/api/roadtrip/1/organiser', {
+      statusCode: 200,
+      body: {},
+    });
+    cy.intercept('GET', '/api/roadtrip/1/emergencydetails/user', {
+      statusCode: 200,
+      body: {
+        emergencyContact: { name: 'Tim', relation: 'Father', phoneNumber: '982 7380 289' },
+        _id: '609770ac85a60252abedd1a5',
+        name: 'Sally Blue',
+        phoneNumber: '83 392 839',
+        user: '608602c6c4155b006f4d3124',
+        __v: 0,
+      },
+    });
+    cy.intercept('GET', '/api/roadtrip/1/emergencydetails', {
+      statusCode: 200,
+      body: [
```

Comment:
This body could be good to place in the fixtures json, then you can access it from fixtures.body (or whatever else you decide to name it), just to make the tests cleaner :) Only if you have time though